packages/core-mobile/app/store/notifications/listeners/listeners.ts

```diff
@@ -86,23 +90,17 @@ export const addNotificationsListeners = (
       setAccount,
       onFcmTokenChange,
       turnOnNotificationsFor,
-      onBalanceChangeNotificationsEnabled
+      onBalanceChangeNotificationsEnabled,
+      turnOnAllNotifications
```

Comment:
I think we should separate the balance subscribe logic from the others. for example, when adding a new account, it makes sense to call subscribe balance for the new address. however, it is not necessary for the other types of notification